test/integration/banListTest.ts

```diff
@@ -192,28 +193,28 @@ describe("Test: Updating the BanList", function () {
     })
 });
 
-describe('Test: We do not respond to recommendations other than m.ban in the banlist', function () {
-    it('Will not respond to a rule that has a different recommendation to m.ban (or the unstable equivalent).', async function () {
+describe('Test: We do not respond to recommendations other than m.ban in the banlist', function() {
+    it('Will not respond to a rule that has a different recommendation to m.ban (or the unstable equivalent).', async function() {
         const mjolnir = config.RUNTIME.client!
         const banListId = await mjolnir.createRoom();
-        const banList = new BanList(banListId, banListId, mjolnir);
-        await createPolicyRule(mjolnir, banListId, RULE_SERVER, 'exmaple.org', '', {recommendation: 'something that is not m.ban'});
+        const banList = new PolicyList(banListId, banListId, mjolnir);
+        await createPolicyRule(mjolnir, banListId, RULE_SERVER, 'exmaple.org', '', { recommendation: 'something that is not m.ban' });
```

Comment:
Could you explain how these formatting changes were spotted? (Did you use another linter or editor extension?) Dealing with a merge conflict in this file and worried about missing some.